src/acceptance/assets/file/policy/policy-with-credential-type.json

```
@@ -0,0 +1,81 @@
+{
+  "credential_type": "x509",
```

```suggestion
  "credential-type": "x509",
```

As it has no relation to the policy json I'm fine with keeping kebab-case to be as aligned to the normal naming of this parameter as possible.